### DISCUSSION TITLE
storage issues

### DISCUSSION THREAD

**Discussion by decenttp** [ltsp/ltsp#631] [Q&A]
Hi,
I have a new LTSP server setup ( Intel(R) Xeon(R) CPU E3-1225 v5 @ 3.30GHz with 16 gigs of ram) 

Ltsp users are currently facing strange issues with the internal 1TM HDD drive that's currently mounted for /home folder for all the LTSP users. Intermittently ltsp users  end up finding blank folders with none of their data visible. Their dats shows up when they reboot and restart the ltsp sessions and wondering what might be causing this and a fix for the same. 

Thanks

**Comment by alkisg** [comment]
Can you upload (1) your ltsp.conf (replace any sensitive information with xxx), and (2) the output of `cat /proc/mounts` from a logged in client?

**Reply by decenttp** [reply]
[mounts.zip](https://github.com/ltsp/ltsp/files/7726334/mounts.zip)

Please find the two files attached.

**Comment by alkisg** [comment]
From what I can see, /home/username is mounted via SSHFS from the server, which is the default for LTSP.
When you say "internal 1TB HDD", do you mean that this disk exists in the server or in the clients?

In general, SSHFS is quite stable, but it doesn't support file locks well. If you're using some application that has trouble with SSHFS, you might want to test NFS /home instead, see the [ltsp nfs man page](https://ltsp.org/man/ltsp-nfs/) for how to set it up.

**Reply by decenttp** [reply]
Disk exists on the server only. I have the OS on a separate SSD and only the /home mounted on a separate 1TB HDD for size. Clients drives are only being used for local swap as suggested by you to resolve the intermittent client freeze issues. I tried to monitor clients to see any specific pattern but could not figure out one.
